openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -58,4 +58,4 @@ <h1>Merge duplicate editions</h1>
 {{ pager() }}
 
 </body>
-</html>
+</html>
```

Comment:
unrelated whitespace change here. I can amend on merge if necessary.

Reply:
fixed this